### ISSUE TITLE
historic=ruins may be used as way

### ISSUE THREAD

**Issue by maxolasersquad** [openstreetmap/id-tagging-schema#1149]
**Describe the bug**
When a way is tagged with `historic=ruins` the issue 'Ruins should be a closed area based on the tag "history=ruins"' is raised. The wiki for [Tag:historic=ruins](https://wiki.openstreetmap.org/wiki/Tag:historic=ruins) explicitly states that ways are valid for this tag.

**To Reproduce**
Create a way and tag it with `historic=ruins`.

**Expected behavior**
No issue should be given for this tagging scheme on ways.

**Screenshots**
![image](https://github.com/openstreetmap/id-tagging-schema/assets/177150/67e2db33-59a3-492c-afbd-44fafc8c26f2)

**Additional context**
[An example way that the editor flags as an issue.](https://www.openstreetmap.org/way/138519891)

**Comment by matkoniecz** [comment]
> According to [taginfo](https://taginfo.openstreetmap.org/tags/historic%3Druins#overview), nearly 3/4 of the usage of this tag is a way.

Note that it includes also closed lines which are areas

**Comment by tyrasd** [comment]
Right, the actual [number](https://dashboard.ohsome.org/#backend=ohsomeApi&groupBy=none&time=2007-10-08T00%3A00%3A00Z%2F2024-02-27T16%3A00Z%2FP1M&measure=count&filter=geometry%3Aline+and+historic%3Druins&bboxes=-180%2C-90%2C180%2C90) is more like 2%, but it's still good to fix this. :blush:
